lib/probes/MemoryProbe.js

```diff
@@ -0,0 +1,37 @@
+var Q = require('q'),
+	util = require('util'),
+	events = require('events'),
+	helpers = require('../helpers'),
+	debug = require('debug')('bp:probes:MemoryProbe');
+
+function MemoryProbe(id) {
+	if (id) {
+		this.id = id;
+	}
+	events.EventEmitter.call(this);
+}
+
+util.inherits(MemoryProbe, events.EventEmitter);
+
+MemoryProbe.prototype.id = 'MemoryProbe';
+
+MemoryProbe.prototype.setup = function(cfg) {
+	this.timerHandle = null;
+};
+
+MemoryProbe.prototype.start = function(browser) {
+	var me = this;
+	this.timerHandle = setInterval(function() {
+		browser.eval('window.performance.memory').then(function(res) {
+			if(res) {
+				me.emit('data', res);
```

Comment:
I was going to suggest that instead of getting this and emiting it back to browser-perf, we could store all these results in the browser, and then finally send it all back, but I realize that accumulating the result in the browser could lead to increased memory, which adds extra noise due to this test runner.